run.sh

```diff
@@ -0,0 +1,57 @@
+# Argument Parsing
+echo "Positional Parameters"
+echo 'Github Project Repository: ' $1
+echo 'Github Project Repository Module Path: ' $2
+
+if [ "$1" == "" ]; then
+    echo "arg1 - github project repository"
+    exit
+elif [ "$2" == "" ]; then
+    echo "arg2 - github project module path"
+    exit
+fi
+
+# Setup Variables
+crnt=`pwd`
+working_dir=`dirname $0`
+project_repository=$1
+project_module_path=${crnt}/$2
+
+# TestRunner
+if [ ! -d "testrunner" ]; then
+    git clone https://github.com/ReedOei/testrunner.git
+fi
+cd testrunner
+mvn install
+cd ..
+
+# Dt-Fixing-Tools
+if [ ! -d "dt-fixing-tools" ]; then
```

Comment:
It shouldn't be necessary to clone the dt-fixing-tools, as presumably users would almost surely have cloned this repo to have access to this script in the first place.

However, it may still be useful to build it for them, if they have not already.

Reply:
This just double checks to make sure the repo has been cloned, in case the user simply grabbed the raw script from github without cloning the repository. If it has, it won't clone dt-fixing-tools